community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/Sargable.scala

```diff
@@ -57,6 +57,16 @@ object AsPropertySeekable {
   }
 }
 
+//This is when dynamic properties are used
+object AsDynamicPropertyNonSeekable {
```

Comment:
If it is just me I'm fine with it, anyone else having an opinion? @systay @boggle ?